Dockerfile

```diff
@@ -44,7 +44,11 @@ RUN (git remote add riscv https://github.com/${GITHUB_REPOSITORY} && \
 FROM v8-riscv as v8-precheck
 RUN git fetch riscv riscv64
 RUN bash /root/commit-msg-check.sh riscv/riscv64 $(git log --format="%H" -n 1)
-RUN git cl format --presubmit && git diff --exit-code
+RUN apt install python-pip
+RUN pip install coverage
+RUN pip install numpy
+RUN pip install mock
```

Comment:
Let's move all of these `pip install` commands up into `v8-base`, after the `apt-get` command.